src/cryptenroll/io.systemd.cryptenroll.policy

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?> <!--*-nxml-*-->
+<!DOCTYPE policyconfig PUBLIC "-//freedesktop//DTD PolicyKit Policy Configuration 1.0//EN"
+        "https://www.freedesktop.org/standards/PolicyKit/1/policyconfig.dtd">
+
+<!--
+  SPDX-License-Identifier: LGPL-2.1-or-later
+
+  This file is part of systemd.
+
+  systemd is free software; you can redistribute it and/or modify it
+  under the terms of the GNU Lesser General Public License as published by
+  the Free Software Foundation; either version 2.1 of the License, or
+  (at your option) any later version.
+-->
+
+<policyconfig>
+
+        <vendor>The systemd Project</vendor>
+        <vendor_url>https://systemd.io</vendor_url>
+
+        <action id="io.systemd.cryptenroll.enroll">
```

Comment:
i'd really split this up, and have distinct actions for pkcs11, tpm2, fido2. admins might want to turn off one mechanism, so maybe split this up?

Reply:
or hmm, maybe not, dunno. could also be passed to the polkit logic via that metainfo dict

Reply:
Yeah I was considering splitting it up. Not sure what the best move here is :shrug:. We could definitely get a nice message out of it

I should also probably use the dict to pass along the device node we're operating on

Another question: the EnrollXXXXX methods imply a wipe, just like the CLI. Wasn't sure how to handle that in polkit. I decided not to (i.e. the various enroll-* polkit actions just also permit a wipe and that's that). Maybe I need to change the wording, or maybe we need to add a `-with-wipe` variants of the various actions. Up to you